src/_tests_/RockIndex.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from "@testing-library/react"
+import RockIndex from "../Pages/RockIndex"
+import mockRocks from "../mockRocks"
+
+
+describe("<RockIndex />", () => {
+  it("renders without crashing", () => {
+    const div = document.createElement("div")
+    render(<RockIndex mockRocks={mockRocks} />, div)
+  })
+ 
+  it("renders rock cards", () => {
+    const div = document.createElement("div")
+    render(<RockIndex rocks={mockRocks} />, div)
+    screen.logTestingPlaygroundURL()
+    // mockRocks.forEach((rock) => {
+    //   const rockName = screen.getByText(rock.name)
+    //   expect(rockName).toBeInTheDocument()
+    // })
+  })
```

Comment:
Looks like you are missing an assertion. Please reach out if you need help. Also remember to remove the debugging tools and commented out code before submitting for PR reviews.